src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MaxLatencySpec.groovy

```diff
@@ -242,6 +242,58 @@ class MaxLatencySpec extends HealthCheckSpecification {
         database.resetCosts()
     }
 
+    @Tidy
+    def "Able to create DEGRADED flow with LATENCY strategy if flowPath > max_latency"() {
+        given: "2 non-overlapping paths with 11 and 15 latency"
+        setLatencyForPaths(11, 15)
+
+        when: "Create a flow, maxLatency 5 and maxLatencyTier2 10"
+        def flow = flowHelperV2.randomFlow(switchPair).tap {
+            allocateProtectedPath = false
+            maxLatency = 9
+            maxLatencyTier2 = 16
+            pathComputationStrategy = PathComputationStrategy.LATENCY.toString()
+        }
+        northboundV2.addFlow(flow)
+
+        then: "Flow is created in DEGRADED state because flowPath doesn't satisfy max_latency value"
```

Comment:
please add `but satisfies max_latency_tier2`, would make the intended behavior very clear